GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleActivity.java

```diff
@@ -13,17 +13,74 @@
  * limitations under the License.
  */
 
-
 package org.gearvrf.sample;
 
-import android.os.Bundle;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
 
 import org.gearvrf.GVRActivity;
+import org.gearvrf.GVRScreenshotCallback;
+import org.gearvrf.utility.Log;
+
+import android.graphics.Bitmap;
+import android.os.Bundle;
+import android.os.Environment;
+import android.view.MotionEvent;
 
 public class SampleActivity extends GVRActivity {
-    @Override
-    protected void onCreate(Bundle icicle) {
-        super.onCreate(icicle);
-        setScript(new SampleCubeScript(), "gvr_note4.xml");
-    }
+	@Override
+	protected void onCreate(Bundle icicle) {
+		super.onCreate(icicle);
+		setScript(new SampleCubeScript(), "gvr_note4.xml");
+	}
+
+	@Override
+	public boolean onTouchEvent(MotionEvent event) {
+		captureScreenLeft(newScreenshotCallback("screenshot_left.png"));
```

Comment:
I _guess_ this is why you put the API in `GVRActivity`, not `GVRContext`, but it still seems to me like the API belongs in `GVRContext` - the `GVRActivity` creates the `GVRScript`, which can always expose the `GVRContext` it gets in `onInit()`, if necessary.